addons/mail/static/src/core/common/composer.js

```diff
@@ -569,9 +569,13 @@ export class Composer extends Component {
     async processMessage(cb) {
         const el = this.ref.el;
         const attachments = this.props.composer.attachments;
-        if (
+        if (attachments.some(({ uploading }) => Boolean(uploading))) {
```

Comment:
I know that's a move but I think the `Boolean` call is useless as the getter always returns a boolean?

Reply:
Yes, I think at some point `uploading` was stored so it could return `undefined`, and we were picking with enforcing boolean rather than truthy/falsy. I will remove it because I like less code for improved readability